### PR TITLE
fix(components/datetime): remove vertical spacing and fix horizontal padding

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[AB#2552087](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2552087)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1662/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1662/storybooks/ag-grid-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1662/storybooks/datetime-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1662/storybooks/lists-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1662/storybooks/tabs-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1662/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1662/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1662/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1662/playground/)

File: apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
+      <label class="sky-control-label"> First name </label>
+      <input
+        formControlName="firstName"
+        type="text"
+        spellcheck="false"
+        class="sky-form-control"
+      />
+    </sky-input-box>
+    <div>
       <sky-date-range-picker
```
Comment (Blackbaud-ErikaMcVey) [inline apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html:13]: this should have a sky-margin-inline-lg on it

File: libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.scss [hunk 2]
```diff
@@ -5,9 +5,11 @@
   display: flex;
 }
 
-.sky-date-range-picker-form-group {
-  &:last-child {
-    margin-bottom: 0;
+@media (max-width: $sky-screen-sm-min) {
+  .sky-date-range-picker-form-group {
+    &:not(.sky-date-range-picker-last-input) {
+      margin-bottom: $sky-space-md;
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.scss]: if you use `var(--sky-margin-stacked-lg)` I think you won't have to specify different styles for modern theme

**Review by Blackbaud-ErikaMcVey — COMMENTED** [review]
spacing within the date range picker itself looks great! I think in the visual tests though its using `sky-margin-stacked-sm` between the multiple date range pickers when it should be using `sky-margin-stacked-lg`

File: apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html (hunk 1 above)
Comment (Blackbaud-SandhyaRajasabeson) [reply apps/playground/src/app/components/datetime/date-range-picker/date-range-picker.component.html:13]: ah yeah i removed it for testing, good catch 😅

**Review by Blackbaud-CoreyArcher — APPROVED** [review]

**Pull request merged**